### ISSUE TITLE
[Engine Request] Namu Wiki

### ISSUE THREAD

**Issue by mystery-z** [searx/searx#3396]
**Working URL to the engine**
[namu wiki](https://namu.wiki/)

**Why do you want to add this engine?**
It's essentially Wikipedia but specifically in Korean (anyone can edit, but requires citations and etc), and generally has a bigger focus on Current/Recent Events that aren't exactly laid out in detail on Wikipedia. 

One advantage of this over normal Korean Wikipedia is that the flow of new information is very fast here, because this has kinda become the go-to site (especially among people under 30), rather than Wikipedia.

**Features of this engine**
It's pretty easy to integrate, while there isn't an API as far as I know, you can always just do "https://namu.wiki/Go?q=" and then put the search term after that. 

Generally pretty fast. 

**How can Searx fetch the information from this engine?**
I don't think this is a proper "API" but you can just do "https://namu.wiki/Go?q=" and then add the search term (as I mentioned above).

**Applicable category of this engine**
General 

**Additional context**
Tl;dr: Pretty popular wiki in korea, as popular as [ko.wikipedia](https://ko.wikipedia.org/wiki/%EC%9C%84%ED%82%A4%EB%B0%B1%EA%B3%BC:%EB%8C%80%EB%AC%B8), if not more popular.
